### PR TITLE
dts:arm:st:h7  Add usart10 device node in stm32h723.dtsi as per #33475

### PR DESCRIPTION
**Pull request by kshitij9192**
In this PR, I have tried to add usart10 device node in stm32h723.dtsi.

The property values for this node was pulled from stm32h723 technical reference manual.

Memory mapped location:
image

RCC register value:
image

Interrupt:
image
image

### CODE REVIEW

**Comment by Kshitij4kk** [comment]
@KwonTae-young

**Comment by manoj153** [comment]
Hi, all this commit looks good.

**Review by erwango — CHANGES_REQUESTED** [review]
Thanks for this addition.
The code change itself is ok, but you need a little change in the commit message to fit the project guideline (and get compliance check scripts happy):
```
dts: add usart10 device node in dts/arm/st/h7/stm32h723.dtsi

USART 10 is available in STM32H723 SoC and related variants.
Fixes #33475

Signed-off-by: Kshitij Kumar <kshitij9192@gmail.com>
```

**Comment by Kshitij4kk** [comment]
Hi @erwango , thanks for the suggestion.

I have update the commit message. Kindly help to have a look when possible.
Thanks for your support.

File: dts/arm/st/h7/stm32h723.dtsi [hunk 1]
```diff
@@ -22,6 +22,15 @@
 			status = "disabled";
 			label = "UART_9";
 		};
+
+		usart10: serial@40011C00 {
```
Comment (KwonTae-young) [inline dts/arm/st/h7/stm32h723.dtsi:26]: I think `C` should be `lowercase`.
Recent commits are shown in lowercase.
https://github.com/zephyrproject-rtos/zephyr/commit/aace0063ecf6e3069494834d4a9f98422dffb12d#diff-c401060329ae8e5b90389bec57dfa4f088d9db73a260b07bf35ea87ef8362650
https://github.com/zephyrproject-rtos/zephyr/commit/8d09a4942834d5a23a81401748a8b9d65b171208#diff-c401060329ae8e5b90389bec57dfa4f088d9db73a260b07bf35ea87ef8362650

**Review by KwonTae-young — CHANGES_REQUESTED** [review]

File: dts/arm/st/h7/stm32h723.dtsi (hunk 1 above)
Comment (manoj153) [reply dts/arm/st/h7/stm32h723.dtsi:26]: Sharp eyes to spot that. Is it part of the device tree or zephyr convention?

Comment (KwonTae-young) [reply dts/arm/st/h7/stm32h723.dtsi:26]: There are [Coding Guidelines](https://docs.zephyrproject.org/latest/contribute/index.html#coding-guidelines).
There is no direct content related to the device tree, but there are a number of commits worth referencing.

**Comment by Kshitij4kk** [comment]
@KwonTae-young , I have changed the node address to lower case character as suggested.
Kindly help to review when possible. 
Thanks for your review.

**Review by erwango — CHANGES_REQUESTED** [review]
Please squash your commits, otherwise, this seems fine.

**Comment by FRASTM** [comment]
This PR has probably to be closed, since the https://github.com/zephyrproject-rtos/zephyr/pull/34591 is merged

**Comment by erwango** [comment]
Fixed (zephyrproject-rtos/zephyr#34591)

**Pull request closed without merging**